gemini/tests/set4/timeline.js

```diff
@@ -8,13 +8,27 @@ var WAIT_TIME = 5000;
 var WAIT_LOAD_TIME = 1000;
 
 gemini.suite('timeline', child => {
-  child
-    .setUrl('/timeline')
-    .before((actions, find) => {
-      actions.waitForElementToShow('.content-area', WAIT_TIME);
-      actions.wait(WAIT_LOAD_TIME);
-    })
-    .setCaptureElements('.content-area')
-    .ignoreElements({ every: '.gemini-ignore' })
-    .capture('default');
+  gemini.suite('static', child => {
+    child
+      .setUrl('/timeline/static')
+      .before((actions, find) => {
+        actions.waitForElementToShow('.content-area', WAIT_TIME);
+        actions.wait(WAIT_LOAD_TIME);
+      })
+      .setCaptureElements('.content-area')
+      .ignoreElements({ every: '.gemini-ignore' })
```

Comment:
ok, I simply implemented the test for the angular component in the same way the one for the static version, I'll change both

Reply:
![ezgif-3-02d9379752e1](https://user-images.githubusercontent.com/3706809/66686472-e34d3380-ec7f-11e9-95fb-c60e7517b2e1.gif)

Tried with
```
.ignoreElements({ every: 'clr-spinner' })
```

but is not stable.

Reply:
It looks like there are two timeline spinners. Can you try `.ignoreElements({every: 'clr-spinner'})`? Or, use the spinner css class selector: `ignoreElements({every: '.spinner'})`.

Reply:
as I said in the previous comment I already trued `.ignoreElements({ every: 'clr-spinner' })` but it failse because of the changing size of the item (this is why I posted the GIF)

Reply:
tried both `.ignoreElements({every: 'clr-spinner'})` and `.ignoreElements({every: '.spinner'})` with the same results.

Since the item is animated the excluded area is always different. I tried to find a way to specify some "margin"/"padding" for the excluded area in the gemini documentation but without success.

As I wrote in my first reply, the original test was made excluding the whole step; maybe the author already tried several solutions

Reply:
Lets' just leave this one out for now it's causing too much trouble. I have some ideas for how to update the tests that may work but I don't want to push that work onto you and this PR> Thanks for trying to make it work.